helpers/wrapped_key.sh

```diff
@@ -26,24 +26,38 @@ secret_name=$3
 project_id=$4
 exe_path=$(dirname $0)
 
+trap 'catch $? $LINENO' EXIT
+catch() {
+  if [ "$1" != "0" ]; then
+    echo "Error $1 occurred on $2"
+    gcloud projects remove-iam-policy-binding $project_id --member=serviceAccount:${terraform_service_account} --role=roles/cloudkms.cryptoOperator
```

Comment:
Good catch! We'll work on it.

Reply:
@erlanderlo 
Are you considering break the terraform deploy in case the gcloud command not be able to remove the role or just print a message as the terraform output? Currently the script is already sending an error message on the screen if the role don't be removed.

Reply:
I could be wrong, but I thought the approach was to avoid any unexpected "surprises."  Will the error message actually result in tf apply stopping saying, "hey there was something wrong you should fix" or leave it up to the user to notice...oh there was some message in the log, but we deployed with success.  Therefore, leaving more privilege to that SA with CryptoOperator.

I will say breaking tf mid deploy can be a mess - leaving tfState file in a less than ideal state.  I would favor on letting deploy finish and adding a better message with additional doc saying "please rerun this command xyz to remove unecessary priv" 

Hope that helps.  And if anything, maybe open a differnent issue (bug type) to fix in a future PR, since the gcloud command failing is likely rare in the catch, right?